$MirOS: ports/games/bnetd/patches/patch-conf_bnetd_conf_in,v 1.1.7.1 2005/03/18 15:46:05 tg Exp $
--- conf/bnetd.conf.in.orig	Fri Mar 29 08:17:18 2002
+++ conf/bnetd.conf.in	Fri Apr 22 20:52:34 2005
@@ -11,12 +11,12 @@
 
 ##############################################################################
 # Privileges to work under                                                   #
-# Use either the names from the corresponding files or numerical id's (with  #
+# Use either the names from the corresponding files or numerical IDs (with   #
 # a leading '#').  If none are specified, the userid will not be changed.    #
 #----------------------------------------------------------------------------#
 
-#effective_user  = games
-#effective_group = games
+effective_user  = _bnetd
+effective_group = _bnetd
 
 # effective_user  = #12
 # effective_group = #20
@@ -47,8 +47,8 @@ mpqfile     = /usr/local/etc/autoupdate
 logfile     = /usr/local/var/bnetd.log
 realmfile   = /usr/local/etc/realm.list
 maildir     = /usr/local/var/bnmail/
+pidfile     = /usr/local/var/bnetd.pid
 versioncheck_file = /usr/local/etc/versioncheck
-#pidfile     = /usr/local/var/bnetd.pid
 
 fortunecmd  = /usr/games/fortune
 
@@ -224,7 +224,7 @@ hide_pass_games = true
 hide_started_games = false
 
 # Should non-permanent channels hidden on the channel list?
-hide_temp_channels = true
+hide_temp_channels = false
 
 # Should the extended /-commands be avaliable? (of course!)
 extra_commands = true
@@ -400,7 +400,6 @@ use_keepalive = false
 
 #                                                                            #
 ##############################################################################
-
 
 ##############################################################################
 # Telnet configuration                                                       #
